image_layers/map-image-layer-sublayer-visibility/src/main/java/com/esri/samples/map_image_layer_sublayer_visibility/MapImageLayerSublayerVisibilitySample.java

```diff
@@ -75,27 +73,39 @@ public void start(Stage stage) {
       controlsVBox.getChildren().addAll(citiesBox, continentsBox, worldBox);
       controlsVBox.getChildren().forEach(c -> ((CheckBox) c).setSelected(true));
 
-      // create a ArcGISMap with the a BasemapType Topographic
+      // create a map view
+      mapView = new MapView();
+
+      // create an ArcGISMap with the topographic basemap and set it to the map view
       ArcGISMap map = new ArcGISMap(Basemap.Type.TOPOGRAPHIC, 48.354406, -99.998267, 2);
+      mapView.setMap(map);
 
       // create a Image Layer with dynamically generated ArcGISMap images
-      ArcGISMapImageLayer imageLayer = new ArcGISMapImageLayer(WORLD_CITIES_SERVICE);
-      imageLayer.setOpacity(0.7f);
+      ArcGISMapImageLayer imageLayer = new ArcGISMapImageLayer("https://sampleserver6.arcgisonline.com/arcgis/rest/services/SampleWorldCities/MapServer");
 
       // add world cities layers as ArcGISMap operational layer
       map.getOperationalLayers().add(imageLayer);
 
-      // set the ArcGISMap to be displayed in this view
-      mapView = new MapView();
-      mapView.setMap(map);
+      // set the image layer's opacity
```

Comment:
I realize there wasn't a comment before, but we should really have one for why we changed the opacity